handlers/handlers.go

```diff
@@ -391,44 +386,40 @@ func readPreviousReleases(w http.ResponseWriter, req *http.Request, cfg *config.
 	template := "previous-releases"
 	urlPath := path.Dir(req.URL.Path)
 	urlQuery := req.URL.Query()
-
 	// check page type
 	pageData, err := zc.GetPageData(ctx, accessToken, collectionID, lang, urlPath+"/latest")
 	if err != nil {
 		log.Error(ctx, "failed to get content type", err)
 		w.WriteHeader(http.StatusNotFound)
 		return
 	}
-	if !slices.Contains(knownPreviouReleaseTypes, pageData.Type) {
+	if !slices.Contains(knownPreviousReleaseTypes, pageData.Type) {
 		err := errors.New("page type doesn't match known list of content types compatible with /previousreleases")
 		log.Error(ctx, "page type isn't compatible with /previousreleases", err)
 		w.WriteHeader(http.StatusNotFound)
 		return
 	}
-
 	// get cached navigation data
 	navigationCache, err := cacheList.Navigation.GetNavigationData(ctx, lang)
 	if err != nil {
 		log.Error(ctx, "failed to get navigation cache for aggregation", err)
 		setStatusCode(w, req, err)
 		return
 	}
-
-	validatedQueryParams, validationErrs := data.ReviewDataAggregationQueryWithParams(ctx, cfg, urlQuery)
+	validatedQueryParams, validationErrs := data.ReviewDataAggregationQueryWithParams(ctx, cfg, urlQuery, urlPath)
 	if len(validationErrs) > 0 {
 		log.Info(ctx, "validation of parameters failed for aggregation", log.Data{
 			"parameters": validationErrs,
 		})
 		// Errors are now mapped to the page model to output feedback to the user rather than
 		// a blank 400 error response.
-		m := mapper.CreatePreviousReleasesPage(cfg, req, rend.NewBasePageModel(), validatedQueryParams, &searchModels.SearchResponse{}, lang, zebedeeCli.HomepageContent{}, "", navigationCache, template, cache.Topic{}, validationErrs, zebedeeCli.PageData{})
+		m, _ := mapper.CreatePreviousReleasesPage(cfg, req, rend.NewBasePageModel(), validatedQueryParams, &searchModels.SearchResponse{}, lang, zebedeeCli.HomepageContent{}, "", navigationCache, template, cache.Topic{}, validationErrs, zebedeeCli.PageData{})
```

Comment:
I don't think this should return an error.

Reply:
👍